contrib/idxd-config-cmake/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+## Utility library for controlling and configuring Intel® In-Memory Analytics Accelerator (Intel® IAA) available on Next Generation Intel® Xeon® Scalable processors.
```

Comment:
The comment reads a bit like an Intel marketing brochure. Maybe we could at least replace the "Next Generation Intel® Xeon® Scalable processors" by the actual first Xeon generation which supports IAA (aka. Sapphire Rapids).

Reply:
Let's just remove this "Next Generation Intel® Xeon® Scalable processors". We don't write Sapphire Rapids because there would be more new CPU model with IAA  supported in future.